_tests_/Itinerary.test.js

```diff
@@ -0,0 +1,15 @@
+/* globals describe it expect */
+const Intinerary = require('../src/Itinerary');
+
+describe('Intinerary constructor function', () => {
+  it('can be instantiated', () => {
+    expect(new Intinerary()).toBeInstanceOf(Object);
+  });
+
+  it('can have ports', () => {
+    const nice = jest.fn();
+    const monaco = jest.fn();
+    const intinerary = new Intinerary([nice, monaco]);
+    expect(intinerary.ports).toEqual([nice, monaco]);
```

Comment:
Minor point but its a common convention to leave a clear line before the start of your assertions at the end of a test
```
    const intinerary = new Intinerary([nice, monaco]);

    expect(intinerary.ports).toEqual([nice, monaco]);
```